.annotation_safe_list.yml

```diff
@@ -8,52 +8,48 @@
 #    ".. choice_annotation:": foo, bar, baz
 
 canvas.HistoricalCanvasEnterpriseCustomerConfiguration:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 consent.HistoricalDataSharingConsent:
-    ".. pii:": "The username field inherited from DataSharingConsent contains PII"
-    ".. pii_types:": username
-    ".. pii_retirement:": consumer_api
+  ".. pii:": "The username field inherited from DataSharingConsent contains PII"
+  ".. pii_types:": username
+  ".. pii_retirement:": consumer_api
 degreed.HistoricalDegreedEnterpriseCustomerConfiguration:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalEnrollmentNotificationEmailTemplate:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalEnterpriseCourseEnrollment:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalEnterpriseCustomer:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalEnterpriseCustomerCatalog:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalEnterpriseCustomerEntitlement:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalEnterpriseAnalyticsUser:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalPendingEnterpriseCustomerUser:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalPendingEnterpriseCustomerAdminUser:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalLicensedEnterpriseCourseEnrollment:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 enterprise.HistoricalPendingEnrollment:
-    ".. no_pii:": ""
+  ".. no_pii:": ""
 moodle.HistoricalMoodleEnterpriseCustomerConfiguration:
-    ".. no_pii:": ""
-enterprise.EnterpriseCourseEntitlement:
```

Comment:
I don't think these are necessary here, I annotated them on the models, but if we need them here let me know so I can comment why.